scripts/create-agent.ts

```diff
@@ -141,10 +140,17 @@ class VargasJRAgentCreator {
   private async createEC2Instance(keyPairName: string): Promise<string> {
     console.log("Creating EC2 instance...");
     
+    const securityGroupId = await findOrCreateSecurityGroup(
+      this.ec2,
+      "vargas-jr-ssh-access",
+      "Security group for VargasJR agent SSH access"
+    );
+    
     const result = await this.ec2.runInstances({
-      ImageId: "ami-0c02fb55956c7d316",
+      ImageId: "resolve:ssm:/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2",
```

Comment:
Instead of resolving a image id, let's use the same ImageId that the production Vargas JR agent uses by dynamically fetching it for now